content/concepts/introduction/protocols/multistream.md

```diff
@@ -1,7 +1,23 @@
 ---
 title: "Multistream Select"
-description: ""
+description: "Multistream Select is used to negotiate the protocol of a connection between two peers in libp2p."
 weight: 26
 ---
 
-Coming soon!
+## Overview
+
+[Multistream Select](https://github.com/multiformats/multistream-select) is a protocol for
+protocol multiplexing that allows for the negotiation of different protocols between two
+entities. It is a specific protocol built on top of the general concept of Multistream.
+It is used to negotiate the protocol of a connection between two peers and to integrate
+new protocols as they become available. Every multistream message is prefixed with a
+header, `/multistream/1.0.0`.
+
+Peers use Multistream Select at various places to select the protocols to use for a
+libp2p connection, including [transports](../../transports/overview.md),
+[security](../../secure-comm/overview.md), and [stream multiplexing](../../multiplex/overview.md)
+as necessary. The [protocol negotiation](../core-abstractions/connections#protocol-negotiation)
+and [upgrade process](../core-abstractions/connections#upgrading-connections) are explained
+further in the [connections document](../core-abstractions/connections.md).
+
```

Comment:
We should add a sentence saying that it's also used to select application protocols spoken on streams.

Reply:
:+1: done with the last 3 commits.